src/components/NavBarOut.js

```diff
@@ -1,56 +1,69 @@
 import React, { useState } from 'react';
 import '../styles/NavBar.css';
+import LogInButton from './LogInButton';
```

Comment:
navbar for abouot and navbar out look exactly the same...consider using reusable components

Reply:
NavBarForAboutPages has two primary differences. its like on the slider goes back to the landing page, and there is not log in button on the about pages nav bar. all of my nav bars will be very similar but have small differences like that. if there is a better way to make it more resuable im fine with it.